### PR TITLE
[Integrations][NewRelic] Fix not handling a failure of getting related entity guid for an alert 

### PR DESCRIPTION
**Pull request by Tankilevitch**
# Description

What -
Why -
How -

## Type of change

Please leave one option from the following and delete the rest:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Non-breaking change (fix of existing functionality that will not change current behavior)
- [ ] Documentation (added/updated documentation)

### CODE REVIEW

File: integrations/newrelic/newrelic_integration/ocean.py [hunk 1]
```diff
@@ -25,9 +25,8 @@ async def resync_entities(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
             return
         # mainly for issues newRelicAlert as it has different resync logic than entities
         if not port_resource_configuration.selector.entity_query_filter:
-            logger.debug(
-                "Skipping resync for kind without entity_query_filter",
-                kind=kind,
+            logger.info(
+                f"Skipping resync for kind without entity_query_filter, kind={kind}",
```
Comment (MPTG94) [inline integrations/newrelic/newrelic_integration/ocean.py:29]:
```suggestion
                f"Skipping resync for kind without entity_query_filter, kind: {kind}",
```

File: integrations/newrelic/.port/resources/port-app-config.yaml [hunk 2]
```diff
@@ -71,3 +52,22 @@ resources:
             error_rate: .apmSummary.errorRate
             response_time_avg: .apmSummary.responseTimeAverage
             instance_count: .apmSummary.instanceCount
+  - kind: newRelicAlert
```
Comment (MPTG94) [inline integrations/newrelic/.port/resources/port-app-config.yaml:55]: Did this actually change? if not then please make sure there is no actual diff to keep clean history

File: integrations/newrelic/newrelic_integration/core/issues.py [hunk 3]
```diff
@@ -56,31 +59,47 @@ async def list_issues(
...
+                        # by the format of .__<type>.entity_guids.[<entity_guid>...]
+                        issue.setdefault(
+                            f"__{queried_entities[entity_guid]}",
+                            {},
+                        ).setdefault("entity_guids", []).append(entity_guid)
+                    except NewRelicNotFoundError:
+                        logger.info(
+                            f"Related entity guid {entity_guid} not found in new relic for issue {issue.get('issueId')}, continuing",
+                        )
+                    except Exception as err:
```
Comment (MPTG94) [inline integrations/newrelic/newrelic_integration/core/issues.py:97]: Will this exception ever be thrown?

**Review by MPTG94 — APPROVED** [review]
Take a look at my small comments, LGTM

File: integrations/newrelic/newrelic_integration/core/issues.py (hunk 3 above)
Comment (Tankilevitch) [reply integrations/newrelic/newrelic_integration/core/issues.py:97]: could be, I don't want to fail no matter what

File: integrations/newrelic/.port/resources/port-app-config.yaml (hunk 2 above)
Comment (Tankilevitch) [reply integrations/newrelic/.port/resources/port-app-config.yaml:55]: I preferred to change the order, it makes more sense to first export the services and then the alerts

**Pull request merged**